### ISSUE TITLE
how do I trigger timer interrupt and entry_trap in murax

### ISSUE THREAD

**Issue by fatalfeel** [SpinalHDL/VexRiscv#207]
usually need a mtvec
https://stackoverflow.com/questions/68614732/problems-using-risc-v-timer-interrupts-and-simulating-with-spike

how to do in murax in C code or .S or inline asm

**Comment by Dolu1990** [comment]
Right, to reduce area usage, as it would have been 32 bits registers ^^
On ice40, that also mean 32 wasted lut.
